ENSAdmin: show primary names and avatars of owners (#381)
namehash/ensnode: apps/ensadmin-next/README.md

You can update `PREFERRED_ENSNODE_URL` environment variable if you wish ENSAdmin to include a given URL as an initial connection option.

> #### RPC URLs for ENS Name Resolution

Reviewer: ```suggestion
#### RPC URLs
```

These will be used for more than just ENS Name Resolution

> ENSAdmin uses [wagmi](https://wagmi.sh/) to resolve primary ENS names for Ethereum addresses. You need to provide RPC URLs for each chain where ENS is deployed:

Reviewer: ```suggestion
ENSAdmin may use RPC URLs for each chain with a "root" ENS Deployment:
```

These will be used by more than just wagmi or for the resolution of primary ENS names.
